lib/home/profile/account/account_settings.dart

```diff
@@ -2,29 +2,9 @@ import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:thc/firebase/user.dart';
 import 'package:thc/home/profile/account/account_field.dart';
+import 'package:thc/home/profile/account/close_account.dart';
 import 'package:thc/home/profile/profile.dart';
-import 'package:thc/main.dart';
 import 'package:thc/utils/navigator.dart';
-import 'package:firebase_auth/firebase_auth.dart';
-
-_showSuccessDialog(BuildContext context) {
-  showDialog(
-    context: context,
-    builder: (_) => AlertDialog(
-      title: const Text('Account Closed'),
-      content: const Text('You have successfully closed your account.'),
-      actions: [
-        ElevatedButton(
-          onPressed: () {
-            // go back to the home page
-            navigator.pushReplacement(const ChooseAnyView());
-          },
-          child: const Text('OK'),
-        ),
-      ],
-    ),
-  );
-}
```

Comment:
The dialogs involved in deleting the user's account have been moved into **close_account.dart**.